website/blog/2021-02-16-64updates.md

```diff
@@ -0,0 +1,203 @@
+- [react-native-splash-screen](https://github.com/crazycodeboy/react-native-splash-screen) ([in PR](https://github.com/crazycodeboy/react-native-splash-screen/pull/514))
+- [react-native-fs](https://github.com/itinance/react-native-fs) ([in PR](https://github.com/itinance/react-native-fs/pull/952))
+
+… and more are coming! You can see which modules we’re tracking on the [Community Module Requests project board.](https://github.com/microsoft/react-native-windows/projects/23)
+
+In addition to the modules that we have recently worked on, you can view and search the complete list of modules that work with Windows by visiting [reactnative.directory](https://reactnative.directory/). The directory catalogues modules that work on the major platforms including iOS, Android, Windows, and macOS and provides metrics around usage, popularity, and more.
+
+You can also check out the [react-native-gallery repo](https://github.com/microsoft/react-native-gallery) to see some of the modules “in action”. We’re using the Gallery as a testbed to verify and showcase modules that work with Windows – we’re adding new samples over time, so check back often!
+
+## 🚧 A module I need doesn't work with Windows yet!
+
+Fret not! We’re continually reassessing and reviewing the set of modules that are important to React Native for Windows devs, so if you need a module that doesn’t yet have a Windows implementation, please [open a new issue on the React Native for Windows GitHub repo.](https://github.com/microsoft/react-native-windows/issues/new/choose)
+
+If you have experience writing Windows native code, you can also use the [new Community module template workflow](https://microsoft.github.io/react-native-windows/docs/next/native-modules-setup) to create a module template for Windows. This is a great way to start using native Windows APIs quickly in custom modules.
+
+## 🤝 Janea Systems
+![janea-logo](assets/janeasystemslogo.png)
+
+Finally, we wanted to highlight our partnership with [Janea Systems](http://www.janeasystems.com/), who has helped provide implementation, consultation, and support for many of the modules listed above. In particular, we wanted to shout out to [@jaimecbernardo](https://github.com/jaimecbernardo), [@joaocgreis](https://github.com/joaocgreis), and [@bzoz](https://github.com/bzoz) from Janea Systems for their help over the last few months.
+
+# Breaking changes
+
+- **Native module thread affinity**
+
+ A bug was discovered which led to custom native modules often being invoked on the UI thread. This was unintentional and has been changed. We recognize that this may break existing modules in cases where affinity was relied upon without explicitly queuing to the UI thread. Please let us know if you are effected by this change, or are using a community module which is affected.

```

Comment:
Might be a good place to link to https://microsoft.github.io/react-native-windows/docs/ireactcontext-api#uidispatcher, which is what you would use to post to the UI thread if your module requires it.